src/Symfony/Component/Yaml/Tag/TaggedValue.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Yaml\Tag;
+
+/**
+ * @author Nicolas Grekas <p@tchwork.com>
+ * @author Guilhem N. <egetick@gmail.com>
+ */
+final class TaggedValue
+{
+    private $value;
+    private $tag;
+
+    /**
+     * @param null|bool|string|int|float|array $value
+     * @param string                           $tag
+     */
+    public function __construct($value, $tag)
+    {
+        $this->value = $value;
+        $this->tag = $tag;
+    }
+
+    /**
+     * @return null|bool|string|int|float|array
```

Comment:
It can't be an object else than `TaggedValue` (`!foo !php/object:...` is not possible). But I'm not sure such complex type hint is needed, i'll go for `mixed`.